gin.go

```diff
@@ -582,6 +582,23 @@ func (engine *Engine) RunQUIC(addr, certFile, keyFile string) (err error) {
 	return
 }
 
+// RunTLSAndQUIC attaches the router to a http.Server and starts listening and serving
+// both TLS/TCP and QUIC connections in parallel requests.
+// It is a shortcut for http3.ListenAndServe(addr, certFile, keyFile, router)
+// Note: this method will block the calling goroutine indefinitely unless an error happens.
+func (engine *Engine) RunTLSAndQUIC(addr, certFile, keyFile string) (err error) {
```

Comment:
maybe we should keep `RunQUIC` func and replace `http3.ListenAndServeQUIC` with `http3.ListenAndServeTLS`